### PR TITLE
[Merged by Bors] - chore(GroupTheory/MonoidLocalization): Change `Localization` to an abbrev of `OreLocalization`.

### PR DESCRIPTION
**Pull request by erdOne**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by erdOne** [comment]
!bench

File: Mathlib/RingTheory/Localization/Basic.lean [hunk 1]
```diff
@@ -911,34 +911,11 @@ instance instUniqueLocalization [Subsingleton R] : Unique (Localization M) where
     Localization.induction_on a fun _ => by
       congr <;> apply Subsingleton.elim
 
-/-- Addition in a ring localization is defined as `⟨a, b⟩ + ⟨c, d⟩ = ⟨b * c + d * a, b * d⟩`.
-
-Should not be confused with `AddLocalization.add`, which is defined as
-`⟨a, b⟩ + ⟨c, d⟩ = ⟨a + c, b + d⟩`.
--/
-protected irreducible_def add (z w : Localization M) : Localization M :=
```
Comment (mattrobball) [inline Mathlib/RingTheory/Localization/Basic.lean:919]: Is this still irreducible wherever it now is?

**Comment by erdOne** [comment]
!bench

File: Mathlib/RingTheory/OreLocalization/Basic.lean [hunk 2]
```diff
@@ -585,6 +645,51 @@ instance : CommMonoid R[S⁻¹] where
 
 end CommMonoid
 
+section Zero
+
+variable {R : Type*} [Monoid R] {S : Submonoid R} [OreSet S] {X : Type*} [Zero X]
+variable [MulAction R X]
+
+private irreducible_def zero (lemma := zero_def') : X[S⁻¹] := 0 /ₒ 1
```
Comment (mattrobball) [inline Mathlib/RingTheory/OreLocalization/Basic.lean:653]: I think we prefer the `irreducible` tag now and then we can use `unseal`/`with_unfolding_all` to access the internals.

**Comment by erdOne** [comment]
!bench

**Comment by erdOne** [comment]
!bench

**Comment by erdOne** [comment]
!bench

**Comment by erdOne** [comment]
!bench

**Comment by erdOne** [comment]
!bench

**Comment by erdOne** [comment]
!bench

**Comment by erdOne** [comment]
!bench

File: Mathlib/Topology/ContinuousFunction/UniqueCFC.lean [hunk 3]
```diff
@@ -265,8 +265,8 @@ lemma toNNReal_mul_add_neg_mul_add_mul_neg_eq (f g : C(X, ℝ)₀) :
     ((f * g).toNNReal + (-f).toNNReal * g.toNNReal + f.toNNReal * (-g).toNNReal) =
     ((-(f * g)).toNNReal + f.toNNReal * g.toNNReal + (-f).toNNReal * (-g).toNNReal) := by
   -- Without this, Lean fails to find the instance in time
-  have : AddHomClass (C(X, ℝ≥0)₀ →⋆ₙₐ[ℝ≥0] C(X, ℝ≥0)) C(X, ℝ≥0)₀ C(X, ℝ≥0) :=
-    SemilinearMapClass.toAddHomClass
+  have : SemilinearMapClass (C(X, ℝ≥0)₀ →⋆ₙₐ[ℝ≥0] C(X, ℝ≥0)) (RingHom.id ℝ≥0)
```
Comment (mattrobball) [inline Mathlib/Topology/ContinuousFunction/UniqueCFC.lean:268]: Are these timeouts?

**Comment by mattrobball** [comment]
!bench

File: Mathlib/Topology/ContinuousFunction/UniqueCFC.lean (hunk 3 above)
Comment (erdOne) [reply Mathlib/Topology/ContinuousFunction/UniqueCFC.lean:268]: Yes.

**Comment by mattrobball** [comment]
bors delegate+

**Comment by mattrobball** [comment]
`Mathlib.Analysis.InnerProductSpace.NormPow` et al should really not be downstream of ore localization :(

**Comment by riccardobrasca** [comment]
What is the status of this?

**Comment by erdOne** [comment]
I ran out of time when trying to fix it yesterday. Will merge it today.

File: Mathlib/RingTheory/Localization/Basic.lean [hunk 4]
```diff
@@ -1000,62 +943,10 @@ theorem mk_list_sum (l : List R) (b : M) : mk l.sum b = (l.map fun a => mk a b).
   map_list_sum (mkAddMonoidHom b) l
 #align localization.mk_list_sum Localization.mk_list_sum
 
-theorem mk_multiset_sum (l : Multiset R) (b : M) : mk l.sum b = (l.map fun a => mk a b).sum :=
-  (mkAddMonoidHom b).map_multiset_sum l
+theorem mk_multiset_sum (l : Multiset R) (b : M) : mk l.sum b = (l.map fun a => mk a b).sum := by
+  convert (mkAddMonoidHom b).map_multiset_sum l
```
[the remark this answers is not in the document]
Comment (erdOne) [reply Mathlib/RingTheory/Localization/Basic.lean]: I removed it and it works fine. Not sure what happened...

**Comment by erdOne** [comment]
bors merge

**Pull request closed without merging**